SonarQube.Client/ISonarQubeService.cs

```diff
@@ -93,5 +93,11 @@ public interface ISonarQubeService
         /// </summary>
         /// <remarks>The method does not check whether the project or hotspot exists or not</remarks>
         Uri GetViewHotspotUrl(string projectKey, string hotspotKey);
+
+        /// <summary>
+        /// Returns the source code for the specified file
+        /// </summary>
+        /// <param name="fileKey">e.g. my_project:src/foo/Bar.php</param>
```

Comment:
The format of the `fileKey` property is a bit ugly. I debated whether to have the caller pass the project key and relative file paths separately, since that is what the parameter appears to be.
However, I decided to stick to with the single parameter to start with. If it's inconvenient to work with from the client side then I'll change it later.

Reply:
I don't think it should be on the caller's side, it's an internal implementation detail of the API. But ok if you want to see how it looks.